backoffice/common/components/Sidebar.tsx

```diff
@@ -182,6 +198,19 @@ const Menu = ({ open, setOpen }: Props) => {
           </h5>
           <div className="py-4">{showTax && <ListGroup data={menu_tax_item_data} />}</div>
 
+          <h5 className="font-weight-black text-center text-white">
+            <span>Inventory</span>
+            <span
+              id={styles.burgerButton}
+              onClick={() => {
+                setShowTax(!showTax);
```

Comment:
maybe you mistyped,they're setShowInventory, showInventory...?

Reply:
I fixed it, thanks!!!